recipes/liquid-dsp/all/conanfile.py

```diff
@@ -0,0 +1,170 @@
+from conans import ConanFile, tools
+from contextlib import contextmanager
+import os
+
+
+class LiquidDspConan(ConanFile):
+    name = "liquid-dsp"
+    description = "Digital signal processing library for software-defined radios "
+    topics = ("conan", "dsp", "sdr", "liquid-dsp")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jgaeddert/liquid-dsp"
+    license = ("MIT",)
+    settings = "os", "arch", "build_type", "compiler"
+    exports_sources = ["generate_link_library.bat", "patches/**"]
+    options = {
+        "shared": [True, False],
+        "simdoverride": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "simdoverride": False,
+    }
+
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _libname(self):
+        if self.settings.os == "Windows":
+            return "libliquid"
+        return "liquid"
```

Comment:
Question: Will this + line 168 cause trouble with generated CMake package names?

Reply:
I found out that nope, nevermind